### PR TITLE
Fix host password update

### PR DESCRIPTION
**Pull request by alphadose**

### CODE REVIEW

**Review by Aceix — APPROVED** [review]

File: logic/hosts.go [hunk 1]
```diff
@@ -160,6 +160,9 @@ func UpdateHostFromClient(newHost, currHost *models.Host) (sendPeerUpdate bool)
 		currHost.EndpointIP = newHost.EndpointIP
 		sendPeerUpdate = true
 	}
+	if newHostPass, err := bcrypt.GenerateFromPassword([]byte(newHost.HostPass), 5); err == nil && string(newHostPass) != currHost.HostPass {
+		currHost.HostPass = string(newHostPass)
+	}
```
Comment (abhishek9686) [inline logic/hosts.go:165]: but why would the password get updated for a host?

Comment (alphadose) [reply logic/hosts.go:165]: cc @mattkasun

Comment (mattkasun) [reply logic/hosts.go:165]: I can't think of a use case for updating the Host password.  If it is required, this implementation is not complete in that the mq dynamic-security is not being updated.

**Review by mattkasun — CHANGES_REQUESTED** [review]
I don't think we should be updating the host password.

**Comment by theguy951357** [comment]
will wait on changes

**Pull request closed without merging**